benchmark/benchmark/bench_nearest_neighbors.py

```diff
@@ -0,0 +1,181 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import time
+from typing import Any, Dict, List, Optional, Union
+
+from pyspark.ml.feature import VectorAssembler
+from pyspark.ml.functions import array_to_vector
+from pyspark.sql import DataFrame, SparkSession
+
+from benchmark.base import BenchmarkBase
+from benchmark.utils import with_benchmark
+
+
+class BenchmarkNearestNeighbors(BenchmarkBase):
+    def _supported_class_params(self) -> Dict[str, Any]:
+        params = {"n_neighbors": 200}
+        return params
+
+    def _add_extra_arguments(self) -> None:
+        self._parser.add_argument(
+            "--no_cache",
+            action="store_true",
+            default=False,
+            help="whether to enable dataframe repartition, cache and cout outside fit function",
+        )
+
+    def run_once(
+        self,
+        spark: SparkSession,
+        train_df: DataFrame,
+        features_col: Union[str, List[str]],
+        transform_df: Optional[DataFrame],
+        label_name: Optional[str],
+    ) -> Dict[str, Any]:
+        """
+        This function evaluates the runtimes of SparkCuml NearestNeighbors and Spark LSH, but
```

Comment:
I forgot to replace "SparkCuml" with "SparkRapidsML".

Reply:
Missed that too... will update.

Reply:
done.